docusaurus-docs/community/design/conda-store Usability Study Report.md

```diff
@@ -0,0 +1,208 @@
+![conda-store-artifacts](https://github.com/conda-incubator/conda-store/assets/98317216/d52bbcd9-37c9-4f4b-b1ab-c4d44b9cb334)
+
+* **Visibility:** Users noted that logs and artifacts are not prominently displayed and can be difficult to access, complicating troubleshooting efforts. They can be made more prominent on the environment page and improve access by repositioning them near the build status or highlighting them through UI design changes.
+
+* **Enhanced Presentation:** A suggestion was made for a richer presentation of logs, including syntax highlighting, which can help users parse and understand the logs more efficiently.
+
+* **Platform-Specific Artifacts:** There was interest in having artifacts that cater to specific platforms, such as offering both platform-dependent and independent versions. The types of artifacts can include platform-specific versions where applicable and ensure they are clearly labeled and described to help users choose the appropriate options for their needs.
+
+* **Sharing Artifacts:** Users shared concerns about the security of sharing artifact files directly, with a preference for sharing lockfiles or customized YAML specs without sensitive data. 
+
+### Errors, Debugging and Documentation
+Errors are inevitable parts of managing software environments. The study highlighted several areas where conda-store could improve its handling and presentation of errors to support users better.
+
+* **Log Parsing:** Users frequently check logs and backend processes to diagnose issues with failed or stuck environments. However, they noted that parsing JSON-formatted logs can be complex and error-prone. Enhancing log readability by implementing prettyprint (`pprint`) for JSON-encoded logs, centralizing error messages to make them more accessible, and adding intuitive error parsing tools directly within the UI will aid troubleshooting.
+
+* **Stuck Environments:** Some users encountered issues with environments that remain stuck in the building state, leading to frustration. To improve this, a mechanism can be developed to diagnose and resolve stuck environments more easily. Additionally, users can be provided with tools to force-stop environment builds or delete environments without navigating complex backend processes. Users also prefer to recreate broken environments rather than troubleshoot. This preference indicates a need for streamlined processes to recreate environments quickly, ensuring minimal disruption to workflow.
+
+* **Documentation Visibility and Clarity:** While documentation was deemed helpful, users often have trouble locating it in the conda-store-ui and rely on external search engines due to a broken link within conda-store UI (**Note**: This has been fixed now: link) . Also, the documentation can be expanded to include more detailed descriptions of API endpoints, common errors, and step-by-step troubleshooting guides to support users in resolving issues independently.
```

Comment:
@pavithraes I just realised I missed linking this, but I cannot find the PR. Could you add the link here?

Reply:
Reworded - it is ok if the link is not here